glTF-BinExporter/RhinoDocGltfConverter.cs

```diff
@@ -202,6 +227,17 @@ int[] GetMaterials(RenderMaterial[] materials, RhinoObject rhinoObject)
             return materialIndices;
         }
 
+        private int GetLayerMaterial(RhinoObject rhinoObject)
+        {
+            if (!layerMaterialIndices.TryGetValue(rhinoObject.Attributes.LayerIndex, out int layerMaterialIndex))
+            {
+                Color4f objectColor = GetObjectColor(rhinoObject);
```

Comment:
Should the layer material be relying on the object color? This won't return the layer color if the objects color source isn't from the layer.

Reply:
#d12d92a4ff07f463ad185d5563aa4b72eb971369